### PR TITLE
Added the code snippet for streaming doc

### LINKED ISSUE #254: docs: typescript streaming
Event streaming for workflow runs is now available in the Typescript SDK version 0.1.27

Example of how to listen for events with a workflow run id here:
https://github.com/hatchet-dev/hatchet-typescript-quickstart/blob/main/simple-examples/example-event-with-results.ts

This affects the following pages:
- [ ] https://docs.hatchet.run/home/features/streaming

### PR DESCRIPTION
**Pull request by badex-ai**
# Description
Added the typescript example using the tab style at the "features/streaming" route of the documentation. I am not so familiar with Python so I am not sure if my Python translation is correct. If there are any changes I should make to allow a merge pls reach out and I would. (This is my first open-source pull request)

Fixes # (issue)
254

## Type of change
Documentation

- [ ] Documentation change (pure documentation change)

## What's Changed
Added code snippet

### CODE REVIEW

**Comment by grutt** [comment]
fixes #254

File: frontend/docs/pages/home/features/streaming.mdx [hunk 1]
```diff
@@ -1,7 +1,63 @@
...
+    listener = hatchet.listener.stream(workflow_run_id)
+
+    print("listening for events")
+    async for event in listener:
+        print("event received", event)
+
+    print("done listening for events")
+
+if __name__ == "__main__":
+asyncio.run(main())
```
Comment (grutt) [inline frontend/docs/pages/home/features/streaming.mdx]: tab

File: frontend/docs/pages/home/features/streaming.mdx [hunk 2]
```diff
@@ -1,7 +1,63 @@
...
+
+  </Tabs.Tab>
+  <Tabs.Tab>
+```typescript filename="resultStreaming.ts" copy
+import Hatchet from "@hatchet-dev/typescript-sdk";
+
+async function main() {
+const hatchet = Hatchet.init();
+
+const workflowRunId = await hatchet.admin.run_workflow("simple-workflow", {
```
Comment (grutt) [inline frontend/docs/pages/home/features/streaming.mdx]: consistent spacing

**Review by grutt — CHANGES_REQUESTED** [review]

File: frontend/docs/pages/home/features/streaming.mdx (hunk 2 above)
Comment (badex-ai) [reply frontend/docs/pages/home/features/streaming.mdx]: @grutt pls check and give feedback

File: frontend/docs/pages/home/features/streaming.mdx (hunk 1 above)
Comment (badex-ai) [reply frontend/docs/pages/home/features/streaming.mdx]: @grutt 
hopefully I have resolved this issue. Pls check and give feedback .

**Pull request closed without merging**